m0kv_metadata_parsing_tool/m0kv_metadata_parsing_tool.sh

```diff
@@ -0,0 +1,428 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+red=`tput setaf 1`
+green=`tput setaf 2`
+reset=`tput sgr0`
+rm base64_encoder_decoder -f /dev/null 2>&1
+make > /dev/null 2>&1
+m0kv_PATH="third_party/motr/utils/m0kv"
```

Comment:
is this tool for dev -environments ?

On CORTX stack, there would not be any path like third_party/..